json-rpc/types/Quantity.md

```diff
@@ -0,0 +1,27 @@
+
+# `Quantity`
+The key words “MUST”, “MUST NOT”, “REQUIRED”, “SHALL”, “SHALL NOT”, “SHOULD”, “SHOULD NOT”, “RECOMMENDED”, “MAY”, and “OPTIONAL” in this document are to be interpreted as described in [RFC-2119](https://www.ietf.org/rfc/rfc2119.txt).
+| Requirement | Description  |
+| ----------- | ------------ |
+| 1   | A `Quantity` value **MUST** be hex-encoded |
+| 2   | A `Quantity` value **MUST** be "0x"-prefixed |
+| 3   | A `Quantity` value **MUST** be expressed using the fewest possible hex digits per byte |
+| 4   | A `Quantity` value **MUST** express zero as "0x0" |
+| 5   | A `Quantity` value **MUST** represent an unsigned integer value |
+| 6   | A `Quantity` value **MUST** represent a 32 byte hex encoded decimal with the leading `0`s stripped |
```

Comment:
Isn't this contradictory wrt 5? I guess the idea behind this is to specify that an implementer should be able to accept 256-bit (32bytes) unsigned ints, isn't it?